about.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>About</title>
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.0/css/all.css" crossorigin="anonymous"/>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="about.css">
+</head>
+<body>
+
+  <header>
+    <div class="main-nav-section">
+      <div class="navbar-container">
+        <ul class="navbar font-lato">
+          <li>
+            <i class="fab fa-facebook-f" aria-hidden="true"></i>
+          </li>
+          <li>
+            <i class="fab fa-twitter" aria-hidden="true"></i>
+          </li>
+          <li>
+          <li>
+            <h4>English</h4>
+          </li>
+          <li>
+            <h4>My Page</h4>
+          </li>
+          <li>
+            <h4>Logout</h4>
+          </li>
+        </ul>
+      </div>
+    </div>
+
+    <div class="secondary-navbar">
+      <div class="secondary-sub-container">
+        <div class="navbar-logo">
+          <a class="font-lato" href="index.html">CCS SUMMER CONCERT <br> Series 2021</a>
+        </div>
+        <!-- open and close icons -->
+        <i class="fas fa-bars fa-3x text-white cursor-pointer" id="openMenu" aria-hidden="true" ></i>
+
+        <nav class="pages-menu" id="mob-nav">
+          <i class="fa fa-times fa-3x text-white" aria-hidden="true" id="closeMenu"></i>
```

Comment:
- [ ] OPTIONAL
To improve the **user experience** it will be good practice to ensure that the _hamburger_ and the _close button_ are behaving as buttons when the mouse hovers on them. We can improve upon this aspect by adding the CSS attribute `cursor: pointer` to the _hamburger_ and the _close buttons on _mouse hover_ events. We can additionally change the color or opacity of these objects during the hover event.